cli/tools/bench/mitata.rs

```diff
@@ -9,6 +9,72 @@
 use crate::colors;
 use std::str::FromStr;
 
+fn avg_to_iter_per_s(time: f64) -> String {
+  let iter_per_s = 1e9 / time;
+  let (decimals, fractional) = into_decimal_and_fractional_parts(iter_per_s);
+  human_readable_decimal_with_fractional(decimals, fractional)
+}
+
+/// Return a tuple representing decimal part of provided float, as well as its
+/// first fractional digit.
+fn into_decimal_and_fractional_parts(num: f64) -> (i64, i64) {
+  let decimal_part = num.floor();
+  let fractional_part = {
+    let decs = ((num - num.floor()) * 10.0).round();
+    if decs == 10.0 {
+      0
+    } else {
+      decs as i64
+    }
+  };
+  (decimal_part as i64, fractional_part)
+}
+
+fn human_readable_decimal_with_fractional(
+  decimal: i64,
+  fractional: i64,
+) -> String {
+  // Sweet one-liner to separate integer by commas from:
+  // https://stackoverflow.com/a/67834588/21759102
+  let fmt_decimal = decimal
+    .to_string()
+    .as_bytes()
+    .rchunks(3)
+    .rev()
+    .map(std::str::from_utf8)
+    .collect::<Result<Vec<&str>, _>>()
+    .unwrap()
+    .join(",");
+
+  format!("{}.{}", fmt_decimal, fractional)
+}
+
+#[test]
+fn test_into_decimal_and_fractional_parts() {
```

Comment:
Not really a fan of tests being so inline like this because it's noisy when scrolling between functions.

Reply:
This module hasn't really been visited since it's was added, but I'll make a note of that and do a drive-by cleanup once I touch something around testing/benchmarking in the future.